scripts/gulp/header.js

```diff
@@ -0,0 +1,80 @@
+/**
+ * Copyright 2013-2015, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+'use strict';
+
+var gutil = require('gulp-util');
+var through = require('through2');
+
+var PLUGIN_NAME = 'header';
+var YEAR = '2015';
+
+var HEADERS = {
+  license: function(pkg, version) {
+    return [
+      '/**',
+      ' * ' + pkg + ' v' + version,
+      ' *',
+      ' * Copyright 2013-' + YEAR + ', Facebook, Inc.',
+      ' * All rights reserved.',
+      ' *',
+      ' * This source code is licensed under the BSD-style license found in the',
+      ' * LICENSE file in the root directory of this source tree. An additional grant',
+      ' * of patent rights can be found in the PATENTS file in the same directory.',
+      ' *',
+      ' */',
+      ''
+    ].join('\n');
```

Comment:
```js
return `
/**
 * ...
 */`.trim()
```

Gross all around, which is part of why I moved to reading the template from disk in React.